### PR TITLE
APIMF-2778: apply technical writers feedback on scheduler doc

### PR DESCRIPTION
**Pull request by arielmirra**
apply technical writers feedback on scheduler doc

### CODE REVIEW

File: docs/custom/scheduler.mdx [hunk 1]
```diff
@@ -38,34 +41,36 @@ ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(30, new Th
...
+    - Return a list of tasks that are waiting to be processed, and destroy the `ExecutorService` immediately
+    - All running threads may not be stopped at the same time
+
+
+Refer to the Java documentation for more information about the [`ScheduledExecutorService`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/concurrent/ScheduledExecutorService.html)
 
-More info about `ScheduledExecutorService` interface [here](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/concurrent/ScheduledExecutorService.html).
 
+## Create and add the `ExecutionEnvironment` in AMF
+When initialized, AMF creates a default `amf.client.environment.Environment`. You can customize this environment  by passing an `amf.client.execution.ExecutionEnvironment` object.
```
Comment (IsaacEldridge) [inline docs/custom/scheduler.mdx:64]:
```suggestion
When initialized, AMF creates a default `amf.client.environment.Environment`. You can customize this environment by passing an `amf.client.execution.ExecutionEnvironment` object.
```

File: docs/custom/scheduler.mdx [hunk 2]
```diff
@@ -3,27 +3,30 @@ id: scheduler
 title: Control threads in AMF
 ---
 
+AMF allows you to schedule and manage the lifespan of threads.
```
Comment (IsaacEldridge) [inline docs/custom/scheduler.mdx:6]: should this be the the "lifespan of control threads"?

**Review by IsaacEldridge — APPROVED** [review]
just two little things; otherwise looks good

File: docs/custom/scheduler.mdx (hunk 2 above)
Comment (arielmirra) [reply docs/custom/scheduler.mdx:6]: nope, the word "control" was used like "manage", AMF allows you to control threads, there is no 'control thread' entity, I've confused you with that one sorry

**Review by IsaacEldridge — APPROVED** [review]

**Pull request merged**